README.md

```diff
@@ -44,6 +44,6 @@ Check out who has already contributed to **redash-ui-tests**
 [first]: https://github.com/mozilla/redash-ui-tests/labels/good%20first%20issue
 [git]: https://git-scm.com/
 [redash help]: https://redash.io/help/
-[redash]: https://github.com/getredash/redash
+[redash]: https://wiki.mozilla.org/Telemetry/Custom_dashboards_with_re:dash
```

Comment:
Hey @jezdez @rafrombrc what do you see as the home of the stmo project? The wiki page, the GitHub fork, the running stmo instance? 🤔

Reply:
This is more up-to-date: https://docs.telemetry.mozilla.org/tools/stmo.html